src/clr-base/src/lib/utils/css-vars/css-vars.ts

```diff
@@ -3,42 +3,23 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
+import { cssVars, variableStore } from './css-vars-ponyfill-fork.esm.js';
+import { exists } from '../helpers/exists';
 
-import IWindow from '../../interfaces/window.interface';
+// exported for tests
+export const cssVarsDefaultConfig = {
+  onlyLegacy: true,
+  updateURLs: false,
+  updateDOM: true,
+  onComplete: function() {
+    if (exists(window, 'ShadyCSS', 'styleDocument')) {
+      (window as any).ShadyCSS.styleDocument(variableStore.dom);
+    }
+  },
+};
 
-declare var window: IWindow;
-
-function hasCssVars(win: IWindow): boolean {
-  return win.hasOwnProperty('cssVars');
+export function runCssVarsPolyfill(config?: any): void {
+  cssVars(config || cssVarsDefaultConfig);
```

Comment:
Minor nit: you can have a default js param 

```
export function runCssVarsPolyfill(config = cssVarsDefaultConfig): void {
   cssVars(config);
}
```

Reply:
yup. good catch!

i'll change that.